helmfile/charts/networkpolicy-common/templates/velero/allow-restic.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.velero.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    {{- include "np.labels" . | nindent 4 }}
+  name: allow-restic
+  namespace: velero
+spec:
+  policyTypes:
+    - Ingress
+    - Egress
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: velero
+      name: restic
+  egress:
```

Comment:
Does velero and restic not talk with each other? Do they just use CRDs?

Reply:
Yeah, I had to test this again just to be sure but no they don't talk directly to each other.